Makefile

```diff
@@ -19,3 +19,13 @@ build:
 .SILENT:
 install-site:
 	yarn
+
+# markdownlint-cli
+# Assumes that you have npm installed
+.PHONY: markdownlint-install
+markdownlint-install:
+	npm install markdownlint-cli
```

Comment:
Perhaps you meant to make use of this target above? However, I think it would be better if the below target says it depends on `node_modules/.bin/markdownlint` and this target says how to get that file.

Reply:
Lets solve this in a later PR.